lrauv_ignition_plugins/worlds/portuguese_ledge.py

```diff
@@ -0,0 +1,17 @@
+# Takes in a template and generates an SDF file.
+
+import em
+import os
+import sys
+
+def main():
+    in_file = 'portuguese_ledge.sdf.em'
+    out_file = 'portuguese_ledge.sdf'
+    template = open(in_file).read()
+    arguments = {}
+    result = em.expand(template, arguments)
+    with open(os.path.join(out_file), 'w') as outfile:
```

Comment:
@chapulina nit: can we remove `os.path.join`?

Reply:
I'm using it now after addressing another comment

Reply:
And now not using anymore because the autogeneration uses absolute paths 🙃